src/runner/CMakeLists.txt

```diff
@@ -25,3 +25,6 @@ target_link_libraries(codegen_func ${RUNNER_LIBS})
 add_executable(pool_runner pool_runner.cpp)
 target_link_libraries(pool_runner ${RUNNER_LIBS})
 
+# TEMP - for dev
+add_executable(covid_runner covid_runner.cpp)
+target_link_libraries(covid_runner ${RUNNER_LIBS})
```

Comment:
I'll eventually delete the `covid_runner` but it's proving very useful in aggressively testing the OpenMP implementation.